### PR TITLE
[W5.7][W14-2] Yu Pei Henry

### PR DESCRIPTION
**Pull request by YuPeiHenry**

### CODE REVIEW

**Comment by keenlok** [comment]
Nice job with the deleteAll command! but you might want to update the HelpCommand so that it shows the user how to use the delete all command when they use the app.

**Comment by YuPeiHenry** [comment]
Good catch Keen Lok! I have updated help command to include help text for the deleteAll command. Thanks!

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (kexiaowen) [inline src/seedu/addressbook/parser/Parser.java:14]: Do not import *. Only import the necessary classes explicitly as stated in our coding standard.

**Review by kexiaowen — APPROVED** [review]
@YuPeiHenry Very good attempt! Keep it up! Pls close this PR after reading my comment.

File: src/seedu/addressbook/parser/Parser.java (hunk 1 above)
Comment (YuPeiHenry) [reply src/seedu/addressbook/parser/Parser.java:14]: Oops, I think IntelliJ combined them as .*. Will note, thanks.

**Pull request closed without merging**